seeds/userSeed.js

```diff
@@ -0,0 +1,22 @@
+const { Users } = require('../server');
```

Comment:
require should be from the models folder, and update to be User singular or the deconstruct won't work